product_docs/docs/eprs/7/03_installation/03_installing_rpm_package/x86_amd64/01_eprs_rhel8_x86.mdx

```diff
@@ -0,0 +1,213 @@
+---
+title: "Installing Replication Server on RHEL 8/OL 8 x86"
+navTitle: "RHEL 8/OL 8"
+---
+
+For each step, you must be logged in as superuser.
+
+To log in as a superuser:
+
+```shell
+sudo su -
+```
+
+In addition, you must have Java Runtime Environment (JRE) version 1.8 or later installed on the hosts where you intend to install any Replication Server component (EPRS Replication Console, publication server, or subscription server). Any Java product such as Oracle Java or OpenJDK may be used.
+
+Follow the directions for your host operating system to install Java runtime.
+
+!!! Note
+    For Advanced Server versions prior to 9.3, a Java runtime is supplied and installed as part of the Advanced Server installation process, however, you must still have pre-installed a separate Java runtime system on your host. The Replication Server installation process does not utilize the Java runtime supplied with Advanced Server.
+
+## Set up the repository
+
+1. To create the repository configuration file, assume superuser privileges, and invoke the following command:
+    
+    ```shell
+    dnf -y install https://yum.enterprisedb.com/edbrepos/edb-repo-latest.noarch.rpm
+    ```
+    
+    In the directory `/etc/yum.repos.d`, the repository configuration file `edb.repo` is created, which a text file is containing a list of EnterpriseDB repositories, each denoted by an entry starting with the text `[repository_name]`.
+
+2. Enable access to the packages in any of these repositories:
+
+    - Using your requested credentials for the EDB Yum Repository, substitute the user name and password for the `<username>:<password>` placeholders of the baseurl parameter.
+    - Change the setting of the enabled parameter to `enabled=1`.
+  
+  For example, to access the server `libs` package from the repository for Advanced Server version 14, enable the following entry:
+
+  ```shell
```

Comment:
Try `ini`

Reply:
I think we might be better off with text. I assume ini should color code the values to the right of the equals sign, but it is just color coding the numbers and not the rest of the text.